lineapy/api/api.py

```diff
@@ -359,32 +377,88 @@ def to_pipeline(
     framework: str = "SCRIPT",
     pipeline_name: Optional[str] = None,
     dependencies: TaskGraphEdge = {},
-    pipeline_dag_config: Optional[AirflowDagConfig] = {},
     output_dir: str = ".",
+    input_parameters: List[str] = [],
+    reuse_pre_computed_artifacts: List[str] = [],
+    pipeline_dag_config: Optional[AirflowDagConfig] = {},
 ) -> Path:
     """
     Writes the pipeline job to a path on disk.
 
-    :param artifacts: List of artifact names to be included in the pipeline.
-    :param framework: 'AIRFLOW' or 'SCRIPT'. Defaults to 'SCRIPT' if not specified.
-    :param pipeline_name: Name of the pipeline.
-    :param dependencies: Task dependencies in graphlib format, e.g., {'B':{'A','C'}}
+    Parameters
+    ----------
+    artifacts: List[str]
+        Names of artifacts to be included in the pipeline.
+
+    framework: str
+        "AIRFLOW" or "SCRIPT". Defaults to "SCRIPT" if not specified.
+
+    pipeline_name: Optional[str]
+        Name of the pipeline.
+
+    dependencies: TaskGraphEdge
+        Task dependencies in graphlib format, e.g., ``{"B": {"A", "C"}}``
```

Comment:
I think we should call out loud that no need to specify dependencies within the same session here.

Reply:
Done!